tardis/io/tests/data/arepo_line_reference_model.csvy

```diff
@@ -0,0 +1,41 @@
+---
+name: csvy_full
+model_density_time_0: 0.00115628 day
+model_isotope_time_0: 0.666016 day
+description: Config file for TARDIS from Arepo snapshot.
+tardis_model_config_version: v1.0
+datatype:
+  fields:
+    -  name: velocity
+       unit: cm/s
+       desc: velocities of shell outer bounderies.
+    -  name: density
+       unit: g/cm^3
+       desc: density of shell.
+    -  name: Ni56
+       desc: fractional Ni56 abundance.
+    -  name: Si28
+       desc: fractional Si28 abundance.
+
+---
+velocity,density,Ni56,Si28
+1.11396e+09,0.12017,0.38561,0.221
+nan,nan,nan,nan
+nan,nan,nan,nan
+nan,nan,nan,nan
+nan,nan,nan,nan
+nan,nan,nan,nan
+nan,nan,nan,nan
+nan,nan,nan,nan
+nan,nan,nan,nan
+nan,nan,nan,nan
```

Comment:
Why are these NaN?

Reply:
When rebinning data with scipy.stats.binned_statistic to a higher resolution, it will insert bins with NaN. This behaviour is intended/ I did not include an error for this since it does not break the export and can be adressed manually later on.